### ISSUE TITLE
FAILED: SqsAckSinkTest.testBatchIgnore

### ISSUE THREAD

**Issue by 2m** [akka/alpakka#913]
```
[error] Test akka.stream.alpakka.sqs.javadsl.SqsAckSinkTest.testBatchIgnore failed: java.lang.IllegalStateException: IO Listener actor terminated unexpectedly for remote endpoint [localhost:0], took 0.129 sec
[error]     at akka.stream.impl.io.ConnectionSourceStage$$anon$2.akka$stream$impl$io$ConnectionSourceStage$$anon$$receive(TcpStages.scala:99)
[error]     at akka.stream.impl.io.ConnectionSourceStage$$anon$2$$anonfun$preStart$1.apply(TcpStages.scala:61)
[error]     at akka.stream.impl.io.ConnectionSourceStage$$anon$2$$anonfun$preStart$1.apply(TcpStages.scala:61)
[error]     at akka.stream.stage.GraphStageLogic$StageActor.internalReceive(GraphStage.scala:230)
[error]     at akka.stream.stage.GraphStageLogic$StageActor$$anonfun$1.apply(GraphStage.scala:198)
[error]     at akka.stream.stage.GraphStageLogic$StageActor$$anonfun$1.apply(GraphStage.scala:198)
[error]     at akka.stream.impl.fusing.GraphInterpreter.runAsyncInput(GraphInterpreter.scala:454)
[error]     at akka.stream.impl.fusing.GraphInterpreterShell$AsyncInput.execute(ActorGraphInterpreter.scala:465)
[error]     at akka.stream.impl.fusing.GraphInterpreterShell.processEvent(ActorGraphInterpreter.scala:560)
[error]     at akka.stream.impl.fusing.ActorGraphInterpreter.akka$stream$impl$fusing$ActorGraphInterpreter$$processEvent(ActorGraphInterpreter.scala:742)
[error]     at akka.stream.impl.fusing.ActorGraphInterpreter$$anonfun$receive$1.applyOrElse(ActorGraphInterpreter.scala:757)
[error]     at akka.actor.Actor$class.aroundReceive(Actor.scala:517)
[error]     at akka.stream.impl.fusing.ActorGraphInterpreter.aroundReceive(ActorGraphInterpreter.scala:667)
[error]     at akka.actor.ActorCell.receiveMessage(ActorCell.scala:590)
[error]     at akka.actor.ActorCell.invoke(ActorCell.scala:559)
```

https://travis-ci.org/akka/alpakka/jobs/371451872

**Comment by randomcoder** [comment]
This _should_ have been fixed in `akka-http-10.1.x`.

https://github.com/akka/akka-http/issues/1837

**Comment by ennru** [comment]
The implementation has changed with AWS client 2.0.
Closing.
